saagieapi/env_vars/env_vars.py

```diff
@@ -415,3 +416,74 @@ def export(self, project_id, output_folder: str, error_folder: Optional[str] = "
             write_error(error_folder, "env_vars", project_id)
             result = False
         return result
+
+    def import_from_json(self, json_file: str, project_id: str = None) -> bool:
+        """Import environment variables from JSON format
+        Parameters
+        ----------
+        json_file : str
+            Path to the JSON file that contains env var information
+        project_id : str, optional
+            Project ID
+        Returns
+        -------
+        bool
+            True if environment variables are imported False otherwise
+        """
+        result = True
+
+        try:
+            with open(json_file, "r") as file:
+                env_var_info = json.load(file)
+        except Exception as e:
+            logging.warning("Cannot open the JSON file %s", json_file)
+            logging.error("Something went wrong %s", e)
+            return False
+
+        try:
+            env_var_id = env_var_info["id"]
+            env_var_name = env_var_info["name"]
+            env_var_scope = env_var_info["scope"]
+            env_var_value = env_var_info["value"]
+            env_var_description = env_var_info["description"]
+            env_var_is_password = env_var_info["isPassword"]
+
```

Comment:
Should we add a test to see if the value of the variable is null ? 

If the value of the variable is null , this function will return the following error: 
`TransportQueryError: {'message': 'Environment variable is invalid', 'extensions': {'value': 
"Environment variable value can't be null", 'classification': 'ValidationError'}}`

Reply:
Sure, I will add a test to put an empty value in the variable "env_var_value" if the value found in the JSON file is null or not found.